src/table/components/TableWrapper.jsx

```diff
@@ -59,14 +69,21 @@ export default function TableWrapper(props) {
   const containerMode = constraints.active ? 'containerOverflowHidden' : 'containerOverflowAuto';
   const paginationHidden = constraints.active && 'paginationHidden';
   const paginationFixedRpp = selectionsAPI.isModal() || rect.width < 550;
-  const announce = useMemo(() => announcementFactory(rootElement, translator), [translator.language]);
+  const announce = announcer || useMemo(() => announcementFactory(rootElement, translator), [translator.language]);
```

Comment:
Since `announcer` function has been created inside of the `<TableWrapper />` component and passed down to the dom tree, there is no way to mock it or using `sinon.stub` in order to track it's calls. 

So the `announce` prop here is only for testing purpose here. 
If you think there is a better way for doing it, please let me know

Reply:
Well, or don't test it. We already test the functionality of the page select in `TablePaginationActions.spec.jsx`. The test you are adding is not really testing much new, mostly adding coverage. I'd say it is fine if we miss those two rows, everything around it is tested

Reply:
but i think there is going to be a problem if we skip these tests, since we only test **the visibility** and  the calls of **`onPageChange`** function only in the `TablePaginationActions.spec.tsx` file.

I think we should have some checks around if we announce to the user in case of changes and **specifically what we are announcing to them.**

these two modified test cases on `TableWrapper.spec.jsx` file, only checks for that and the `TablePaginationActions.spec.jsx` file does not even checks for announcements.

we should prevent the possibility of if someone tries to remove/modifies the announce part of the code for pagination stuff by accident, and we can achieve it by these two test cases because (firstly if someone tries too, the test cases will break and secondly) we don't check for announcements of pagination somewhere else in our codebase.

i think it's better for us to keep them

Reply:
Good point. Even though there is no real logic in these functions, it is still important that we do the correct announcement. Keep them but make a comment in the `props` deconstructing that this property should only be used for testing

Reply:
✅ Done!